### DISCUSSION TITLE
Is it possible to paste youtube urls in the player directly?

### DISCUSSION THREAD

**Discussion by rp1231** [422658476/MPV-EASY-Player#48] [Q&A【问答】] [answered]
I would like to paste a youtube url into mpv-easy-player to play the video.
Is that possible?
Currently I can't figure out how to open the youtube url through the gui as well...
Cant seem to find an option called "open url".

I have placed the youtube-dl exe in the root directory of the easy player.
Do I need to do anything else?
Am I missing something or is this feature not available in easy player?

**Comment by 422658476** [comment, marked as answer]
Online videos need to analyze the link to get the correct video address, which is why there is no such function as opening the link.
If you already have the real address of the online video, you only need to save the video address to a txt file, and change the suffix of the txt file to .m3u, and double-click the file to play.

You can refer to the steps on how to stream online TV:

https://github.com/422658476/MPV-EASY-Player    below these texts ： 如何播放获取的在线视频链接

After solving the above problems, you just need to learn how to use youtube-dl.exe to get the real video address, which may require you to check the documentation of youtube-dl.exe
